task/02-numbers-tasks.js

```diff
@@ -181,7 +191,18 @@ function getParallelipidedDiagonal(a,b,c) {
  *   1678, 3  => 2000
  */
 function roundToPowerOfTen(num, pow) {
-    throw new Error('Not implemented');
+    const roundTo = to => Math.round(num / to) * to;
+
+    switch (pow) {
+        case 1:
```

Comment:
It's a hack =) . The solutions will work only for `pow` up to 3. Please try to implement more general solution.